js/main.js

```diff
@@ -26,7 +26,14 @@ export function run(pid, canEdit, jsonAssetUrls) {
   new Vue({
     el: '.vizr-container',
     components: { Vizr },
```

Comment:
The root component is the only one with a template string (all the others are single-file components processed by `vue-loader`), which is why the template compiler was required at runtime. This equivalent render function allows removing the compiler from our bundle.

This change could also be made to phenotypr-body.